src/controllers/fireblocks/fireblocks.controller.js

```diff
@@ -69,23 +69,13 @@ async function startFireblocksWallet(req, res) {
     return;
   }
 
-  const xpubMap = config.xpubs || [];
-  const xPub = xpubMap[xpubId];
-
-  if (!xPub) {
-    console.error('Error starting wallet because xpub-id is referring to a wallet that is not configured.');
-    res.send({
-      success: false,
-      message: `xpub-id ${xpubId} is invalid.`,
-    });
-    return;
-  }
-
   // Checking that the fireblocks config is valid.
   const fireblocksClient = fireblocksService.startClient();
   try {
+    // Here we get the m/44/280/0/0/0 address pubkey from fireblocks
```

Comment:
:star2: I like this comment! It's a good reminder of the small diference to our usual hardened path of derivation